process/README.md

```diff
@@ -22,8 +22,16 @@ flowchart TD
 The process is designed to be flexible to enable a Project to move in and out of a Working Group as deemed appropriate by the TAC.
 
 # II. Lifecycle
+<img align="top" src="https://github.com/ossf/tac/blob/main/process/TI-lifecycle-stages.png">
 
-Working Groups and Projects have a different lifecycle as defined below:
+Technical Initiatives (TIs) include Working Groups (WGs), Special Interest Groups (SIGs) and Projects (both code and specificiations). All TIs follow a common lifecyle, with 4 stages: 
```

Comment:
Maybe we could merge these descriptions / abbreviations with "I. Overview" and just use the abbreviations here?